src/Catalyst.Core.Lib.Tests/UnitTests/Cryptography/CmsTest.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security;
+using System.Text;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Options;
+using Catalyst.Core.Modules.Dfs;
+using Catalyst.Core.Modules.Keystore;
+using Catalyst.TestUtils;
+using Xunit;
+
+namespace Catalyst.Core.Lib.Tests.UnitTests.Cryptography
+{
+    public class CmsTest
+    {
+        private readonly KeyChain keyChain;
+
+        public CmsTest()
+        {
+            keyChain = new KeyChain
+            {
+                Options = new DfsOptions().KeyChain
+            };
+            var securePassword = new SecureString();
+
+            foreach (char c in "mypassword")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)